JavaFXUserInterface/src/main/resources/AssetTypeList.fxml

```diff
@@ -9,12 +9,11 @@
 <?import javafx.scene.control.TextField?>
 <?import javafx.scene.layout.AnchorPane?>
 <?import javafx.scene.layout.FlowPane?>
-<?import javafx.scene.text.Font?>
 <?import javafx.scene.text.Text?>
 <?import javafx.scene.control.TableView?>
 <?import javafx.scene.control.TableColumn?>
 
```

Comment:
There is 5 unresolved fx:id in this file
* search
* assetsTypeTabPane
* thumbnailTab
* listTab
* assetTypesListPane

Any effect?

Reply:
done